ui/src/main/java/com/doordeck/sdk/common/utils/UuidUtils.kt

```diff
@@ -0,0 +1,14 @@
+package com.doordeck.sdk.common.utils
+
+import java.util.UUID
+
+object UuidUtils {
+    fun isUUID(id: String): Boolean {
```

Comment:
Can we keep UUID uppercase? It's somewhat mixed in the code but I think uppercase is the Java convention